### PR TITLE
Inline registration

### PR DESCRIPTION
**Pull request by irdan**
@hitesh96db was able to address all of the issues outlined [here](https://github.com/privly/privly-applications/pull/180#issuecomment-78156797) a few months ago.  I merged the changes into gsoc-develop and updated the name of a local storage key.

### CODE REVIEW

**Comment by irdan** [comment]
We should look in to why the tests didn't run.  Travis says the test [passed](https://travis-ci.org/privly/privly-applications/builds/76817343), but they don't appear to have [run](https://travis-ci.org/irdan/privly-applications/builds/76817248).

**Comment by smcgregor** [comment]
Tests don't run on pull requests because PRs don't have access to the secure environmental variables of Travis.

**Comment by irdan** [comment]
Oh, right. I forgot about [that](http://docs.travis-ci.com/user/pull-requests/#Security-Restrictions-when-testing-Pull-Requests). Perhaps we can set up a webhook to write a comment that links to the travis build on the account of the PR submitter.

**Comment by irdan** [comment]
I just did a manual comparison between this [branch](https://travis-ci.org/irdan/privly-applications/builds/76817248) and the latest build of [gsoc-develop](https://travis-ci.org/privly/privly-applications/builds/76562594) and it appears they have the same errors.  It's probably just a coincidence but I did notice that the gsoc-develop had one more [error](https://travis-ci.org/privly/privly-applications/builds/76562594#L456) than this PR.

**Comment by smcgregor** [comment]
Running [this build](https://travis-ci.org/smcgregor/privly-applications/builds/77072553) from my fork now. gsoc-develop isn't passing at the moment because a whole lot of new tests were introduced and developed for single platforms. It shouldn't take too long to eliminate the test failures for the other platforms because they largely result from one platform being particularly strict compared to the others.

**Comment by smcgregor** [comment]
None of these build errors are from your PR.

This is ready to go once you add a "Check your email for account information" message. It isn't clear what the next step should be.

File: Pages/js/first_run.js [hunk 1]
```diff
@@ -8,6 +8,48 @@
...
+    }
+  },
+
+  /**
+   * Tell the user their registration was submitted.
+   */
+  pendingRegistration: function() {
+    $("#messages").html("<strong>Thanks!</strong> If your email isn't already in our " +
+      "database you should receive an email shortly.");
+    $("#messages").show();
```
Comment (irdan) [inline Pages/js/first_run.js]: @smcgregor This message shows at the top of the screen after submitting the email.  Did you miss it? If so it should probably change to be more obvious.

**Comment by irdan** [comment]
@smcgregor I moved the feedback to directly under the button on the modal.

**Comment by smcgregor** [comment]
I made a few changes in #292. Closing this PR in view of that one.

**Pull request closed without merging**